docs/shadow_config_spec.md

```diff
@@ -472,6 +473,13 @@ Type: Bool
 
 Use seccomp to trap syscalls.
 
+#### `experimental.use_strace_logging`
+
+Default: false  
+Type: Bool
+
+Log syscalls for each process.
```

Comment:
Probably worth mentioning the file name here

Reply:
Added.